native/cocos/core/assets/EffectAsset.h

```diff
@@ -404,7 +404,7 @@ struct IBlockInfo {
 };
 
 struct ISamplerTextureInfo {
-    int32_t binding{-1};
+    uint32_t binding{0};
```

Comment:
binding has to be 0xFFFFFFFF, which is the default (null) value.

Reply:
Yep, seems reasonable.

Reply:
fixed